docs/courses/interactive_modeling.rst

```diff
@@ -0,0 +1,102 @@
+.. _interactive_modeling:
+
+Interactive Modeling
+====================
+
+Physical System
+---------------
+
+Giant axon from the squid: Loligo pealei
+
+Conceptual Model
+----------------
+
+Hodgkin-Huxley cable equations
+++++++++++++++++++++++++++++++
+
+.. image:: img/hh.gif
+    :align: center
+
+Simulation
+----------
+
+Computational implementation of the conceptual model
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+We *could* implement this model in Python:
+
+.. code::
+    python
+
+    from neuron import h, gui
+
+    axon = h.Section(name='axon')
+    axon.L = 2e4
+    axon.diam = 100
+    axon.nseg = 43
+    axon.insert('hh')
```

Comment:
This is correct as-is, but our latest style guidance is for people to write:

```python
axon.insert(h.hh)
```

instead.